src/Employer/UnitTests/EmployerProvideFeedback/Controllers/HomeControllerTests.cs

```diff
@@ -114,5 +96,65 @@ public async void SessionSurvey_Exists_ShouldNotCreateNewSurvey()
             // Assert
             _sessionServiceMock.Verify(mock => mock.Set(uniqueCode.ToString(), It.IsAny<SurveyModel>()), Times.Never);
         }
+
+        private DefaultHttpContext GetHttpContext()
+        {
+            var _context = new DefaultHttpContext()
+            {
+                User = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+                {
+                    new Claim(ClaimTypes.NameIdentifier, GetSurveyModel().UserRef.ToString()),
+                }))
+            };
+            return _context;
+        }
+
+        private SurveyModel GetSurveyModel()
+        {
+            var _surveyModel = new SurveyModel()
+            {
+                UserRef = Guid.NewGuid()
+            };
+            return _surveyModel;
```

Comment:
This can also be one-lined:
```suggestion
            return new SurveyModel()
            {
                UserRef = Guid.NewGuid()
            };
```

Reply:
done